lib/flutter_sound.dart

```diff
@@ -83,11 +83,13 @@ class FlutterSound {
     }
   }
 
-  Future<String> startRecorder(String uri) async {
+  Future<String> startRecorder(String uri, {int sampleRate = 44100, int numChannels = 2}) async {
```

Comment:
Could you make another `method` something like `startRecorderIOS` in order to show that it is only `iOS` compatible?

Reply:
The `PR` has been given on `android` with same feature in #29. I'll resolve this for now.